core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -76,7 +76,7 @@ object TransactionCoordinator {
   }
 
   private def initTransactionMetadata(txnMetadata: TxnTransitMetadata): InitProducerIdResult = {
-    InitProducerIdResult(txnMetadata.producerId, txnMetadata.producerEpoch, Errors.NONE)
+    InitProducerIdResult(txnMetadata.producerId(), txnMetadata.producerEpoch(), Errors.NONE)
```

Comment:
In scala, it can call function without `()`. Could you remove these changes? Thanks.